src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -60,7 +60,7 @@ public function __construct(ContainerBuilder $container, PhpFileLoader $loader,
         $this->container->loadFromExtension($namespace, static::processValue($config));
     }
 
-    final public function import(string $resource, string $type = null, bool $ignoreErrors = false)
+    final public function import(string $resource, string $type = null, int $ignoreErrors = PhpFileLoader::IGNORE_NONE)
```

Comment:
changing the typehint from int to bool is a BC break. We need the BC layer.

Reply:
I have removed the type hint to fix the BC break. As this is final method, it should be ok if some send there some boolean value.